backend/functions/lib/adapters/gateways/dxtr/offices.dxtr.gateway.ts

```diff
@@ -14,6 +14,11 @@ import { buildOfficeCode, getOfficeName } from '../../../use-cases/offices/offic
 
 const MODULE_NAME = 'OFFICES-GATEWAY';
 
+// Remove invalid divisions at the gateway rather than forcing the
+// more important use case code to include logic to remove them.
+const INVALID_DIVISION_CODES = ['070', '990', '991', '992', '993', '994', '995', '996', '999'];
```

Comment:
Curious to know where we got this list.

Reply:
From our analysis of divisions exported from DXTR. Some are obvious. Then 070 is for Islip. We have more to come in the future as the analysis is completed.